internal/context/workspace/domain/workflow/utils.go

```diff
@@ -0,0 +1,9 @@
+package workflow
+
+// SupportedLanguages get supported workflow languages
+func SupportedLanguages() []string {
```

Comment:
I think a property would be just fine? The getter is unnecessary here, and putting it in `utils` is more confusing.

Reply:
done.